test_transcode.py

```diff
@@ -48,7 +51,6 @@ def test_transcode(transcription_id = "keanu.mp3"):
             )
 
             assert len(updates) >= 3
-            assert updates[0].percent_done == 0
```

Comment:
this one also seems highly cpu specific (already >20 when I run it)

Reply:
yes these are flappy